### DISCUSSION TITLE
无板岩矿物和粗矿的本地化译名

### DISCUSSION THREAD

**Discussion by ShangguanXi** [IndustrialCraftMC/IndustrialCraft2#74] [【漏洞反馈】Bugs Report]
![8BIB344@M X~WC2OUVVL$D](https://github.com/IndustrialCraftMC/IndustrialCraft2/assets/78731793/e4355971-9bad-4a31-a527-4f9af5b9fdef)

**Comment by AbbottMc** [comment]
这个得等旧版语言文件完全完成迁移后才会加入

**Comment by AbbottMc** [comment]
已迁移
